pycontrails/models/cocipgrid/cocip_time_handling.py

```diff
@@ -139,18 +139,26 @@ def attach_timedict(self) -> None:
         self.timedict = {t1: (source_time >= t1) & (source_time < t2) for t1, t2 in zipped}
 
     def _check_met_rad_time(self, tmin: pd.Timestamp, tmax: pd.Timestamp) -> None:
-        if self.met.data["time"].min() > tmin or self.met.data["time"].max() < tmax:
-            warnings.warn(
-                "Parameter 'met' is too short in the time dimension. "
-                "Include additional time in 'met' or reduce 'max_age' parameter. "
-                f"Model start time: {tmin} Model end time: {tmax}"
-            )
-        if self.rad.data["time"].min() > tmin or self.rad.data["time"].max() < tmax:
-            warnings.warn(
-                "Parameter 'rad' is too short in the time dimension. "
-                "Include additional time in 'rad' or reduce 'max_age' parameter."
-                f"Model start time: {tmin} Model end time: {tmax}"
-            )
+        """Warn if meteorology data doesn't cover a required time range.
+
+        Parameters
+        ----------
+        tmin: pd.Timestamp
+            Start of required time range
+        tmax:pd.Timestamp
+            End of required time range
+        """
+        met_tmin = pd.to_datetime(self.met.data["time"].min().values)
+        met_tmax = pd.to_datetime(self.met.data["time"].max().values)
+        rad_tmin = pd.to_datetime(self.rad.data["time"].min().values)
+        rad_tmax = pd.to_datetime(self.rad.data["time"].max().values)
```

Comment:
It might be a hair more performant to do something like:

```python
met_time = met.data["time"].values
met_tmin = pd.to_datetime(met_time.min())
met_tmax = pd.to_datetime(met_time.max())
...
```